Source/Scene/TimeDynamicPointCloud.js

```diff
@@ -0,0 +1,554 @@
+define([
+        '../Core/Check',
+        '../Core/combine',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/getTimestamp',
+        '../Core/JulianDate',
+        '../Core/Math',
+        '../Core/Matrix4',
+        '../Core/Resource',
+        './ClippingPlaneCollection',
+        './PointCloud',
+        './PointCloudEyeDomeLighting',
+        './PointCloudShading',
+        './SceneMode',
+        './ShadowMode'
+    ], function(
+        Check,
+        combine,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        getTimestamp,
+        JulianDate,
+        CesiumMath,
+        Matrix4,
+        Resource,
+        ClippingPlaneCollection,
+        PointCloud,
+        PointCloudEyeDomeLighting,
+        PointCloudShading,
+        SceneMode,
+        ShadowMode) {
+    'use strict';
+
+    /**
+     * Provides functionality for playback of time-dynamic point cloud data.
```

Comment:
Style-wise I think "Provides functionality" is empty - could say that about anything.